contrib/st2/opensds/actions/list_volume.py

```diff
@@ -18,7 +18,9 @@
 
 
 class ListVolumeAction(Action):
-    def run(self, ipaddr="", port="", projectid=""):
+    def run(self, ipaddr="", port="", token="", projectid=""):
         url = "http://"+ipaddr+":"+port+"/v1beta/"+projectid+"/block/volumes"
-        r = requests.get(url=url)
+        headers = {'x-auth-token': token}
+        r = requests.get(url=url, headers=headers)
+        print(r.text)
```

Comment:
Are these prints part of results?

Reply:
Yes. List Volume action output to stdout

Reply:
Ok